tests/test_barostat.py

```diff
@@ -185,12 +182,24 @@ def test_barostat_is_deterministic():
         seed,
     )
 
-    ctxt = custom_ops.Context(coords, v_0, host_box, integrator_impl, u_impls, barostat=baro)
+    ctxt = custom_ops.Context(coords, v_0, host_box, integrator.impl(), u_impls, barostat=baro)
```

Comment:
ooh - is this safe now? That's exciting

Reply:
As of https://github.com/proteneer/timemachine/commit/5a1ddfaba3e396a71240d3dea10c55b0813f47ea we don't have to manage wrapped objects the way we use to.